src/server/game/Server/Packets/CharacterPackets.h

```diff
@@ -43,6 +44,17 @@ namespace WorldPackets
 
             bool ShowHelm = false;
         };
+
+        class LoginVerifyWorld final : public ServerPacket
+        {
+        public:
+            LoginVerifyWorld() : ServerPacket(SMSG_LOGIN_VERIFY_WORLD, 4 + 4 * 4) { }
+
+            WorldPacket const* Write() override;
+
+            int32 MapID = -1;
+            Position Pos;
```

Comment:
use `TaggedPosition<Position::XYZO>` here and remove `PositionXYZOStream()` in implementation